resources/assets/ts/vedst-scripts.ts

```diff
@@ -238,20 +237,19 @@ $( document ).ready( function() {
         $('#toggle-shift-time').text(translate('shiftTime'));
 
         // Apply saved preferences from local storage on pageload
-        if(typeof(Storage) !== "undefined") 
+        if(typeof(Storage) !== "undefined")
         {
-            if (localStorage.shiftTime == "show") 
-            {   
-                $('.shift-time').removeClass("hide"); 
+            if (localStorage.getItem("shiftTime") == "show")
+            {
+                $('.entry-time').removeClass("hide");
```

Comment:
Good question, I don't know to be honest. Probably has something to do with the merge back from develop to this branch? 

I'll take a look if there are other lines that were affected.